tools/dts/mpfs_icicle.dts

```diff
@@ -463,9 +463,10 @@
 		};
 	};
 
-	timer0: timer@20125000 {
```

Comment:
Do we really have to drop the "timer0" name here?

Reply:
Nope. Undone.